code/modules/lighting/lighting_system.dm

```diff
@@ -22,18 +24,16 @@
 	Known Issues/TODO:
 		Shuttles still do not have support for dynamic lighting (I hope to fix this at some point) -probably trivial now
 		No directional lighting support. (prototype looked ugly)
-		Allow lights to be weaker than 'cap' radius
 		Colored lights
+		Normalize lumcounts to be from 0 to 1 instead of 0 to 10 so LIGHTING_CALC is unnecessary
 */
 
 #define LIGHTING_CIRCULAR 1									//Comment this out to use old square lighting effects.
 #define LIGHTING_LAYER 15									//Drawing layer for lighting
-#define LIGHTING_CAP 10										//The lumcount level at which alpha is 0 and we're fully lit.
-#define LIGHTING_CAP_FRAC (255/LIGHTING_CAP)				//A precal'd variable we'll use in turf/redraw_lighting()
+#define LIGHTING_CAP 10										//The lumcount level at which we're fully lit.
+#define LIGHTING_CALC(value) (value/LIGHTING_CAP)
```

Comment:
You can do what remie said here too, `#define LIGHTING_CALC_CONST (1/LIGHTING_CAP)` (BYOND should calculate this constant at compile-time hopefully) and then `value*LIGHTING_CALC_CONST`.